register-form/src/components/RegForm.vue

```diff
@@ -0,0 +1,186 @@
+      <img class="radio-img" :src="img.src" :alt="img.val"> 
+    </label>
+    <br>
+    <span class="errorAvatar" v-show="!valids.avatar & !firstTime">*Must chose an avatar.</span>
+    <br>
+
+    <input type="button" class="submit" value="Submit" @click="submitForm()">
+    <br><br>
+
+  </div>
+</template>
+
+<script>
+export default {
+  name: 'RegForm',
+  data(){
+    return {
+
+      inputs: {firstName: "", lastName: "", gender: "", 
+                  email: "", phoneNum: "", avatar: ""},
+
+      images: [{src: require("..\\assets\\elk.svg"), val: "Elk"}, 
+              {src: require("..\\assets\\Gorilla.svg"), val: "Gorilla"},
+              {src: require("..\\assets\\snake.svg"), val: "Snake"}],
+
+      firstTime: true
+    }
+  },
+  computed:{
+
+    isFirstNameValid: function(){
+      return this.inputs.firstName.length > 0
+    },
+    isLastNameValid: function(){
+      return this.inputs.lastName.length > 0
+    },
+    isGenderValid: function(){
+      return this.inputs.gender.length > 0
+    },
+    isEmailValid: function(){
+      let regexEmail =/^\w+([.-]?\w+)*@\w+([.-]?\w+)*(\.\w{2,3})+$/;
+      if (this.inputs.email.match(regexEmail)) {
+        return true; 
+      } 
+      return false; 
+    },
+    isPhoneNumberValid: function(){
+      let phoneno = /^\d{10}$/;
+      if(this.inputs.phoneNum.match(phoneno)){
+        return true
+      }
+      return false
+    },
+
+    isAvatarValid: function(){
+      return this.inputs.avatar.length > 0
+    },
+
+    valids: function(){
+            return {firstName: this.isFirstNameValid, lastName: this.isLastNameValid,
+               gender: this.isGenderValid, email: this.isEmailValid, 
+               phoneNum: this.isPhoneNumberValid, avatar: this.isAvatarValid}
+    },
+    
+  },
+  methods:{
+    submitForm: function(){
+      this.firstTime = false
+      if(this.allValid())
+        console.log(this.inputs)
+    },
+    allValid : function(){
+      let areAllValid = true
+      for(let key in this.valids){
+        areAllValid = areAllValid & this.valids[key]
+      }
+      return areAllValid
+    }
+  }
+}
+</script>
+
+<style scoped>
+
+  input, select{
+    margin: center;
+    background-color: rgba(145, 79, 120, 0.2);
+  }
+  .invalid{
+    /* display: inline-block; */
+      background-color: rgba(255,0,0,0.2);
+  }
+
+  [type=radio] { 
+    display: none;
+  }
+  
+  /* IMAGE STYLES */
+  [type=radio] + img {
+    cursor: pointer;
+  }
+   
+  .invalidAvatar {
+    /* box-sizing: border-box;
+     */
+    border: none;
+    outline: 1px solid rgba(255,0,0,0.5);
+  }
+
+  /* CHECKED STYLES */
+  [type=radio]:checked + img {
+    outline: 2px solid rgba(145, 79, 120, 0.5);
+  }
+  .radio-img{
+    width:150px;
+    height:150px;
+  }
+
+  .submit{
+    font-family: Comic Sans MS, Comic Sans, cursive;
+    color: rgb(19, 5, 66);
+    background: rgba(145, 79, 120, 0.5);
+    cursor: pointer;
+  }
+
+  .submit:active{
```

Comment:
just FYI, you can write `:active{...}` inside the already existing class for `.submit`...
just different syntax that uses scopes and IMO (In My Opinion) more readable